pom.xml

```diff
@@ -69,7 +69,7 @@
 
     <properties>
         <!-- Versions of third party dependencies -->
-        <aspect-meta-model-version>1.0.0</aspect-meta-model-version>
+        <aspect-meta-model-version>2.0.0</aspect-meta-model-version>
```

Comment:
As this version is not on Maven Central yet, this is the last thing blocking the PR from merging. This probably needs to be changed to "2.0.0-M1" or similar when the BAMM 2.0.0 Milestone is released.